The ScreenName function removed in this patch was removed from the "network"
module recently.

TODO: Upstream this.

diff --git a/src/include/rdp/dialogs.ycp b/src/include/rdp/dialogs.ycp
index 2189d55..7fe38e7 100644
--- a/src/include/rdp/dialogs.ycp
+++ b/src/include/rdp/dialogs.ycp
@@ -23,8 +23,6 @@ include "network/routines.ycp";
  */
 define symbol RemoteMainDialog() {
 
-    ScreenName("rdp");
-
     /* Ramote Administration dialog caption */
     string caption = _("Remote Administration");
 
